environments/prod/prod.tfvars

```diff
@@ -2990,13 +2990,38 @@ frontends = [
       {
         match_variable = "RequestCookieNames"
         operator       = "Equals"
-        selector       = "cookie_setting"
+        selector       = "cookie_settings"
```

Comment:
you were right first time - it is 'cookie_setting' (sorry I made a mistake in the slack message)